### DISCUSSION TITLE
Can't figure out the issue | code=exited, status=203/EXEC

### DISCUSSION THREAD

**Discussion by blanc42** [DNSCrypt/dnscrypt-proxy#2200] [Q&A]
```
dnscrypt-proxy.service - Encrypted/authenticated DNS proxy
     Loaded: loaded (/etc/systemd/system/dnscrypt-proxy.service; enabled; vendor preset: disabled)
     Active: activating (auto-restart) (Result: exit-code) since Sun 2022-09-18 12:22:50 IST; 4s ago
    Process: 320918 ExecStart=/opt/dnscrypt-proxy/dnscrypt-proxy -config dnscrypt-proxy.toml (code=exited, status=203/EXEC)
   Main PID: 320918 (code=exited, status=203/EXEC)
```
the dnscrypt-proxy service throws the above error
It works as a standalone but throws an error when run as a service

**Comment by jedisct1** [comment]
Before installing the service, at [step 5](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-linux#step-5-tweak-the-configuration-file), did the `./dnscrypt-proxy` command start without retuning any error?

**Reply by blanc42** [reply]
Nope,
It is running perfectly as a standalone application

It throws the following error when I tried to resolve though
(Currently I am running it on port 53000)
![Screenshot from 2022-09-19 18-41-28](https://user-images.githubusercontent.com/85779539/191026248-e84ed33e-3370-4cf5-9579-c89bf7ace52f.png)

**Reply by jedisct1** [reply]
But just before trying to use it with `-resolve`, your terminal shows `Stopped` ?
